sass/chosen.scss

```diff
@@ -5,140 +5,130 @@
-    .group-name{
-      margin-right: 4px;
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      font-weight: normal;
-      color: #999999;
-      &:after {
-        content: ":";
-        padding-left: 2px;
-        vertical-align: top;
-      }
+  /* include_group_label_in_selected */
+  .group-name {
+    font-weight: bold;
+    margin-right: 4px;
+    &:after {
+      content: ":";
     }
   }
 }
 /* @end */
 
 /* @group Single Chosen */
-.chosen-container-single{
+.chosen-container-single {
   .chosen-single {
-    position: relative;
+    background: $chosen-background;
+    border: 1px solid $chosen-border;
+    border-radius: $chosen-radius;
+    color: $chosen-text;
     display: block;
+    height: $chosen-height;
+    line-height: $chosen-height - 2px;
     overflow: hidden;
-    padding: 0 0 0 8px;
-    height: 25px;
-    border: 1px solid #aaa;
-    border-radius: 5px;
-    background-color: #fff;
-    @include background(linear-gradient(#fff 20%, #f6f6f6 50%, #eee 52%, #f4f4f4 100%));
-    background-clip: padding-box;
-    box-shadow: 0 0 3px #fff inset, 0 1px 1px rgba(#000,.1);
-    color: #444;
+    padding: 0 25px 0 8px;
+    position: relative;
     text-decoration: none;
-    white-space: nowrap;
-    line-height: 24px;
-  }
-  .chosen-default {
-    color: #999;
-  }
-  .chosen-single span {
-    display: block;
-    overflow: hidden;
-    margin-right: 26px;
     text-overflow: ellipsis;
     white-space: nowrap;
-  }
-  .chosen-single-with-deselect span {
-    margin-right: 38px;
-  }
-  .chosen-single abbr {
-    position: absolute;
-    top: 6px;
-    right: 26px;
-    display: block;
-    width: 12px;
-    height: 12px;
-    background: $chosen-sprite -42px 1px no-repeat;
-    font-size: 1px;
-    &:hover {
-      background-position: -42px -10px;
+    &.chosen-default {
+      color: #999;
     }
-  }
-  &.chosen-disabled .chosen-single abbr:hover {
-    background-position: -42px -10px;
-  }
-  .chosen-single div {
-    position: absolute;
-    top: 0;
-    right: 0;
-    display: block;
-    width: 18px;
-    height: 100%;
-    b {
-      display: block;
-      width: 100%;
+    /* down arrow */
+    > div {
+      background: $chosen-sprite no-repeat 0 3px;
       height: 100%;
-      background: $chosen-sprite no-repeat 0px 2px;
+      position: absolute;
+      right: 0;
+      top: 0;
+      width: 18px;
+    }
+    /* deselect icon */
+    &.chosen-single-with-deselect {
+      padding-right: 45px;
+      > abbr {
+        background: $chosen-sprite no-repeat -42px 1px;
+        height: 12px;
+        position: absolute;
+        right: 28px;
+        top: 7px;
+        width: 12px;
+        &:hover {
+          background-position: -42px -10px;
+        }
+      }
     }
   }
+  /* active */
+  &.chosen-container-active .chosen-single {
+    border-color: $chosen-highlight;
+    outline: auto 5px -webkit-focus-ring-color;
+    outline-offset: -2px;
```

Comment:
These outline properties will work on Mac, not PC. But that's also why the `border-color` is there.